resources/devbox/macos/macos.sh

```diff
@@ -150,6 +150,9 @@ BREW_IOS="swiftlint carthage"
 BREW_MACOS="carthage cocoapods"
 BREW_ANDROID="openjdk@8 android-sdk android-studio ant gradle maven"
 
+# Turn on verbosity
+set -x
```

Comment:
Can move this to the top with the other Bash flags (ll. 3-4)

Reply:
I thought about that, but I think it will add too much noise… I placed this line, right where I thought the real actions start taking place.